releasenotes/jmri4.5.6.shtml

```diff
@@ -417,6 +417,54 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=is%3Aclosed&q=milest
 
    <h4>Web Access</h4>
         <ul>
+            <li></li>
```

Comment:
I would be glad to do that, but I don't know exactly what "revert the additions" means.  I will resubmit with all the empty placeholders removed.  If that is incorrect, please advise.
Thanks, sorry i messed it up.
Pete C